magefile.go

```diff
@@ -79,14 +79,22 @@ const (
 	cloudImageTmpl = "docker.elastic.co/observability-ci/elastic-agent:%s"
 )
 
-// Aliases for commands required by master makefile
-var Aliases = map[string]interface{}{
-	"build": Build.All,
-	"demo":  Demo.Enroll,
-}
-var errNoManifest = errors.New("missing ManifestURL environment variable")
-var errNoAgentDropPath = errors.New("missing AGENT_DROP_PATH environment variable")
-var errAtLeastOnePlatform = errors.New("elastic-agent package is expected to build at least one platform package")
+var (
+	// Aliases for commands required by master makefile
+	Aliases = map[string]interface{}{
+		"build": Build.All,
+		"demo":  Demo.Enroll,
+	}
+
+	errNoManifest         = errors.New("missing ManifestURL environment variable")
+	errNoAgentDropPath    = errors.New("missing AGENT_DROP_PATH environment variable")
+	errAtLeastOnePlatform = errors.New("elastic-agent package is expected to build at least one platform package")
+
+	// goIntegTestTimeout is the timeout passed to each instance of 'go test' used in integration tests.
+	goIntegTestTimeout = 2 * time.Hour
+	// goProvisionAndTestTimeout is the timeout used for both provisioning and running tests.
+	goProvisionAndTestTimeout = goIntegTestTimeout + 30*time.Minute
```

Comment:
Scientifically chosen by looking at the total time of the integration test stage for recent builds. It frequently goes over 2h so I couldn't also use 2h for this.

* 2h9m https://buildkite.com/elastic/elastic-agent/builds/7928#018e6376-e786-4ba4-b2eb-fc521ed64ae7
* 2h7m https://buildkite.com/elastic/elastic-agent/builds/7938#018e6a11-a2db-466f-8b8f-548febe75ddb
* 1h53m https://buildkite.com/elastic/elastic-agent/builds/7916

We could probably shorten this but I want to avoid false positives.